lib/API/auth.dart

```diff
@@ -0,0 +1,67 @@
+import 'dart:convert';
+import 'package:app/model/user.dart';
+import 'package:http/http.dart' as http;
+
+class Auth {
+  static String accessToken = "";
+  static int userID = 0;
+  static User user = User();
+  static List<User> allUsers = [];
+
+  // Function to login
+  static Future<bool> login(var basicAuth) async {
+    var url = Uri.parse('http://161.35.99.225/api/v1/auth/login');
```

Comment:
When we use the same value in multiple places, we should put it in a variable to easily deal with it. 
we can write `String baseUrl = 'http://161.35.99.225/api/v1/';` and use the variable everywhere.